### PR TITLE
Initial commit of recipe for malamute (a 0MQ broker) version 1.0

### PR DESCRIPTION
**Pull request by jimklimov**
Requires a git-HEAD libzmq-4.2.0 installed in order to build and run malamute since 2014 (see e.g. https://github.com/OpenIndiana/oi-userland/pull/1814 for the sufficient libzmq bleeding edge version) and a czmq installation

### CODE REVIEW

File: components/library/zeromq/malamute/Makefile [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2014 Alexander Pyhalov.  All rights reserved.
```
Comment (pyhalov) [inline components/library/zeromq/malamute/Makefile]: Remove my copyright

File: components/library/zeromq/malamute/malamute.p5m [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2014 Alexander Pyhalov.  All rights reserved.
```
Comment (pyhalov) [inline components/library/zeromq/malamute/malamute.p5m]: same goes here

**Comment by pyhalov** [comment]
Add specific LDFLAGS so that at build time we look at fixed libzmq.so version

**Comment by jimklimov** [comment]
@pyhalov : revising were old PRs got stuck - how can I "add specific LDFLAGS so that at build time we look at fixed libzmq.so version"? :-)

Also, would such trick help libglew vs. vboxogl?

**Comment by pyhalov** [comment]
Something like  LDFLAGS+=-L/usr/zeromq-3.2/lib ?

**Comment by jimklimov** [comment]
Ah, just that? ;) I thought you meant a way to specify a particular soname.

I think we only use (and codevelop) malamute @work in conjunction with
bleeding-edge zeromq, not sure (anyone checked that) it works with older
releases.

## 

Typos courtesy of GMail on my Samsung Android

17.07.2016 7:36 пользователь "Alexander Pyhalov" notifications@github.com
написал:

**Comment by xen0l** [comment]
I am closing this PR as there hasn't been any activity lately.

**Pull request closed without merging**